src/sw-template.js

```diff
@@ -0,0 +1,97 @@
+if ('function' === typeof importScripts) {
+    importScripts(
+      'https://storage.googleapis.com/workbox-cdn/releases/3.5.0/workbox-sw.js'
+    );
+    
+    /* global workbox */
+    if (workbox) {
+      
+      console.log('Workbox for SFF is loaded');
+  
+      /* injection point for manifest files.  */
+      workbox.precaching.precacheAndRoute([]);
+      
+      /* custom cache rules */
+      workbox.routing.registerNavigationRoute('/index.html', {
+        blacklist: [/^\/_/, /\/[^\/]+\.[^\/]+$/],
+      });
+      
+      /* 
+         Custom cache rules - app images 
+         Cache all image types.
+         Get images from the cache first.  
+         Fetch new images every 30 days so they are not stale - image update are NOT frequent. 
+       */
+      workbox.routing.registerRoute(
+        /\.(?:png|gif|jpg|jpeg|svg)$/,
+        workbox.strategies.cacheFirst({
+          cacheName: 'sf-images',
+          plugins: [
+            new workbox.expiration.Plugin({
+              maxEntries: 60,
+              maxAgeSeconds: 30 * 24 * 60 * 60, // 30 Days
+            })
+          ],
+        })
+      );
+
+      /* 
+         Custom cache rules - json 
+         Cache .geojson files.
+         Fetch garagesale.geojson from the network first if possible (as they are updated frequently).
+         Fallback to the cached garagesale.geojson if disconnected from the internet.
+       */
+      workbox.routing.registerRoute(
+        new RegExp('.*\.json'),
```

Comment:
👍

Reply:
Oops - I see the references to garagesale.geojson in the comments.  This has been cleaned up in 40aff086d801db7c1aeac766d342fa687f1ed02d